Engines/Wine/QuickScript/InstallerScript/script.js

```diff
@@ -102,7 +96,7 @@ InstallerScript.prototype.go = function() {
         .prefix(wine.prefix())
         .search(this._executable)
         .arguments(this._executableArgs)
-        .miniature([this._type, this._category, this._name])
+        .miniature(this.miniature().get())
```

Comment:
Is it ensured at this point that `this.miniature()` is not `empty`?

Reply:
Technically not. However, this can only happen if `getApplication` in `QuickScript` fails. As `getApplication` is called with IDs which have been passed from Java, this is practically impossible.

Reply:
I've thought about it again. It could happen that the `ApplicationDTO` itself does not have a miniature. We do not have this case currently but it might happen. I've updated the code to handle this.